.eslintrc

```diff
@@ -2,8 +2,10 @@
   "extends": "airbnb",
   "rules": {
     "import/extensions": "off",
+    "object-curly-newline": "off",
     "prefer-destructuring": "off",
     "react/destructuring-assignment": "off",
     "react/jsx-filename-extension": "off",
+    "react/jsx-one-expression-per-line": "off"
```

Comment:
We probably don't want to keep adding to this file, but these rules are pretty annoying.